participant/app/routes/about.tsx

```diff
@@ -0,0 +1,33 @@
+import { Alert } from "@trussworks/react-uswds";
+import { Button } from "@trussworks/react-uswds";
+import React from "react";
+import { Trans, useTranslation } from "react-i18next";
+
+export default function Index() {
+  const { t } = useTranslation();
+  const listProcessKeys: string[] = ["answer", "upload", "appointment"];
+
+  return (
+    <div className="measure-6">
+      <h1>{t("About.title")}</h1>
+      <ol className="usa-process-list">
+        {listProcessKeys.map((key: string) => (
+          <li className="usa-process-list__item" key={key}>
+            <h2 className="usa-process-list__heading">
+              <Trans i18nKey={`About.${key}Header`} />
+            </h2>
+            <p className="margin-top-1">
+              <Trans i18nKey={`About.${key}`} />
+            </p>
+          </li>
+        ))}
+      </ol>
```

Comment:
We should be using Truss components here, not using the classes to imitate the components..

https://trussworks.github.io/react-uswds/?path=/docs/components-processlist--process-list-default
There's a `<ProcessList>` component, so we could do

```suggestion
      <ProcessList>
        {listProcessKeys.map((key: string) => (
          <ProcessListItem key={key}>
            <ProcessListHeading type="h2">
              <Trans i18nKey={`About.${key}Header`} />
            </ProcessListHeading>
            <p className="margin-top-1">
              <Trans i18nKey={`About.${key}`} />
            </p>
          </ProcessListItem>
        ))}
      </ProcessList>
```

Reply:
Ah thanks, I was looking at the eligibility screener as an example